tests/testing_meshpy.py

```diff
@@ -300,7 +300,9 @@ def create_comments_in_solid(self, full_import):
         # Convert the solid mesh to meshpy objects.
         mpy.import_mesh_full = full_import
 
-        solid_file = os.path.join(testing_input, "test_meshpy_comments_in_solid.dat")
+        solid_file = os.path.join(
+            testing_input, "test_meshpy_comments_in_solid_initial.dat"
```

Comment:
This is changed, because otherwise the "new" reference file would have the same name, right?

Reply:
Yes exactly - I actually thought too long about the name - we could also take something like `base` or `original`. If you'r fine with `initial` I would merge

Reply:
`initial` is fine with me. I just wanted to double check. Let's merge this!